src/third_party/effective_tld_names/README.drp

```diff
@@ -1,6 +1,6 @@
 URL: http://publicsuffix.org/
      https://publicsuffix.org/list/effective_tld_names.dat
-Version: Downloaded 2/17/2015 from
+Version: Downloaded 1/12/2017 from
          https://publicsuffix.org/list/effective_tld_names.dat
 License: MPL 1.1
```

Comment:
Note that this License: bit is no longer correct, IIRC -- I think it's MPL2 now? May be a good time to correct it